strapi-react-app/react-frontend/package.json

```diff
@@ -30,5 +30,23 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "husky": "^4.2.5",
+    "lint-staged": "^10.2.6",
+    "typescript": "^3.9.3"
+  },
+  "lint-staged": {
+    "src/**/*.{js,jsx}": [
+      "eslint . --fix"
+    ],
+    "src/**/*.scss": [
```

Comment:
have we settled on using scss for styling? Its a pretty old tech in React terms :)

Reply:
yep! Let's  cross that bridge when we get to it. It's best to say as uncluttered as possible. I'm terrible for this.